### ISSUE TITLE
Wallet shows as EMPTY since 3.2.5 Update

### ISSUE THREAD

**Issue by TheVoiceJM** [Berry-Pool/nami-wallet#594]
Since the update on June 4 my Nami Wallet browser extension for Chrome shows the wallet with a zero balance. (Pool shows all is well) Unsure if I'm able to do tx's successfully so I'm not attempting any, effectively locking my funds. Hoping the issue is resolved soon.

**Comment by jujubee789** [comment]
I am having the same issue. Is this being worked on? I tried to restore my wallet on a different computer with the seed phrase and it does the same thing.

**Comment by cordellmac** [comment]
Same issue as well here, I was in a bit of a panic but hopefully my assets are safe.

**Comment by gunchozack** [comment]
I too am in a bit of a panic because I have loads to do with my ADA today…… are our assets safe? Are our funds safe? What’s going on?

**Comment by TheVoiceJM** [comment]
> I too am in a bit of a panic because I have loads to do with my ADA today…… are our assets safe? Are our funds safe? What’s going on?

Yes. If you check the wallet on Pool you'll see everything is there. They may need to put out another update, but everything should be safe. Totally sucks having it locked down, tho. Very frustrating. You may be able to restore it in CC/Eternl and use it that way.

**Comment by gunchozack** [comment]
Ok so what if I don’t have the seed phrase on me? Does that mean I won’t be able to get my funds and assets back? Or should I jus wait for a fix??

**Comment by cordellmac** [comment]
If you don't have the seed phrase, I would 100% wait for a fix.  If you reset, you'll need that seed phrase.

**Comment by cordellmac** [comment]
> Ok so what if I don’t have the seed phrase on me? Does that mean I won’t be able to get my funds and assets back? Or should I jus wait for a fix??

If you don't have the seed phrase, I would 100% wait for a fix. If you reset, you'll need that seed phrase.

**Comment by gunchozack** [comment]
Ok thank you guys for calming me down abit I’ll wait for that fix….. so many mints that I am going to miss on but it’s all good….. thank you again guys…. I’ll jump back on here if anything changes

**Comment by TheVoiceJM** [comment]
I've seen a number of people complaining in the Cornucopias server about it, so it should be widespread enough that they're aware of the issue. At least that's my HOPE. lol

**Comment by gunchozack** [comment]
I think I’ll run with that hope of yours…. It’s better than my dark cloud rn telling me I have lost it all 😂

**Comment by TheVoiceJM** [comment]
Good news! Just got a response from Alessandro (Nami creator) on Twitter. He didn't seem to be aware of it. I responded to him. Hopefully our interaction will get the ball rolling!

**Comment by alessandrokonrad** [comment]
Hey guys, the problem is I'm not facing any of these issues. Things work as before for me. 
What OS are you running: Mac, Window, Linux? And do you use google chrome or brave?

**Comment by gunchozack** [comment]
I use chrome and I am on Windows…….

**Comment by TheVoiceJM** [comment]
Chrome and Windows as well.

**Comment by TheVoiceJM** [comment]
Should we try uninstalling and restore the wallet?

**Comment by gunchozack** [comment]
Someone tried that and it dint work so I don’t know…..

**Comment by gunchozack** [comment]
So I tried to send Ada to my Binance wallet and it worked….. I’ll can confirm that this is jus a bug

**Comment by cordellmac** [comment]
> Hey guys, the problem is I'm not facing any of these issues. Things work as before for me. What OS are you running: Mac, Window, Linux? And do you use google chrome or brave?

Windows 11 here. Chrome Version 102.0.5005.63 (Official Build) (64-bit).  After resetting the wallet Nami has found my stake pool, and the tx's are there as well. No ADA or assets however.

**Comment by alessandrokonrad** [comment]
> > Hey guys, the problem is I'm not facing any of these issues. Things work as before for me. What OS are you running: Mac, Window, Linux? And do you use google chrome or brave?
> 
> Windows 11 here. Chrome Version 102.0.5005.63 (Official Build) (64-bit). After resetting the wallet Nami has found my stake pool, and the tx's are there as well. No ADA or assets however.

If you go to the send tab. Are you able to see your assets there and make transactions?

**Comment by TheVoiceJM** [comment]
I'm on Win 10 and Chrome 102.0.5005.63

**Comment by cordellmac** [comment]
> > > Hey guys, the problem is I'm not facing any of these issues. Things work as before for me. What OS are you running: Mac, Window, Linux? And do you use google chrome or brave?
> > 
> > 
> > Windows 11 here. Chrome Version 102.0.5005.63 (Official Build) (64-bit). After resetting the wallet Nami has found my stake pool, and the tx's are there as well. No ADA or assets however.
> 
> If you go to the send tab. Are you able to see your assets there and make transactions?

Yes, From Send I can see assets and potentially make transactions. Looks like a UI bug.

**Comment by TheVoiceJM** [comment]
> > > Hey guys, the problem is I'm not facing any of these issues. Things work as before for me. What OS are you running: Mac, Window, Linux? And do you use google chrome or brave?
> > 
> > 
> > Windows 11 here. Chrome Version 102.0.5005.63 (Official Build) (64-bit). After resetting the wallet Nami has found my stake pool, and the tx's are there as well. No ADA or assets however.
> 
> If you go to the send tab. Are you able to see your assets there and make transactions?

It appears so. Tx's seem to work, but the wallet displays as empty when you open it.

**Comment by TheVoiceJM** [comment]
Asking around on discord it appears most are using Win/Chrome. Some are reporting Win/Brave.

**Comment by alessandrokonrad** [comment]
That's so weird. I haven't changed anything and as I said I don't face these issues. So I'm not sure where this UI bug suddenly comes from. But I try to investigate. However there is nothing really to worry about as long as you have your seed phrase and see the correct address in the wallet.

**Comment by TheVoiceJM** [comment]
> That's so weird. I haven't changed anything and as I said I don't face these issues. So I'm not sure where this UI bug suddenly comes from. But I try to investigate. However there is nothing really to worry about as long as you have your seed phrase and see the correct address in the wallet.

I just feel better knowing you're aware of it. lol Appreciate the response and hope things work out easily!

**Comment by cordellmac** [comment]
> That's so weird. I haven't changed anything and as I said I don't face these issues. So I'm not sure where this UI bug suddenly comes from. But I try to investigate. However there is nothing really to worry about as long as you have your seed phrase and see the correct address in the wallet.

Appreciate your assistance regardless, puts my mind and I'm sure others at ease knowing its merely a display issue.  any whitelisted sites still still all assets and ADA.  Its not ideal but if I'm not mistaken the wallet is fully functional.

**Comment by heffy09** [comment]
happy
![NamiDisplayIssue](https://user-images.githubusercontent.com/106889948/172026257-52f72ca8-056b-47ac-8bde-d436dcc60152.PNG)
 i found this thread, because im having the same issue where amounts dont show

**Comment by heffy09** [comment]
UPDATE:   re-entering my collateral seems to have fixed the issue??

**Comment by cordellmac** [comment]
> happy ![NamiDisplayIssue](https://user-images.githubusercontent.com/106889948/172026257-52f72ca8-056b-47ac-8bde-d436dcc60152.PNG) i found this thread, because im having the same issue where amounts dont show

Read above, check your send tab and leave as much info as you can.  OS/Browser etc.

**Comment by cordellmac** [comment]
> UPDATE: re-entering my collateral seems to have fixed the issue??

lol.. really? :P if so Kudos, I have to step out for a bit so I'll let others confirm.

**Comment by heffy09** [comment]
> 
yes all figures and amounts are now displayed as they should be. I had a feeling the UI needed some sort of refresh, and removed and re-entered my collateral and upon update for tx of fee for collateral my display is normal and restored. Weird lol

**Comment by TheVoiceJM** [comment]
Took me a little... lol Tried to add collateral but failed a couple times. Finally added. Still zero. Removed collateral...BING...We're back in business! Added collateral back and everything is good. Thanks!!!

**Comment by heffy09** [comment]
> 

sweet! glad it worked for you as well. I just got lucky trying to think of something to refresh it lol

**Comment by alessandrokonrad** [comment]
So here is what I think has happened. Nami uses blockfrost to retrieve data from the chain. After new transactions are detected in Nami, an update event is triggered and the balance on the main screen is updated. I assume blockfrost had some short outage while your balance was updated, that's why a few of you saw balance 0 on the main screen, but the rest was working properly actually. I think it was a coincidence that the update I pushed made it look like this was the issue. This update didn't change anything as mentioned, only updated some text.
